Corona-Warn-App/schemas/de.rki.coronawarnapp.eventregistration.storage.TraceLocationDatabase/1.json

```diff
@@ -62,6 +68,12 @@
             "affinity": "INTEGER",
             "notNull": false
           },
+          {
+            "fieldPath": "byteRepresentation",
```

Comment:
can we rename that? I thought is bytes for the checkIn itself. I think you mean trace location bytes, right?
Maybe `traceLocationBytes`?

Reply:
renamed it